packages/mobile/src/fees/actions.ts

```diff
@@ -0,0 +1,35 @@
+export enum Actions {
+  UPDATE_DEFAULT_FEE = 'FEES/UPDATE_DEFAULT_FEE',
+  DEFAULT_FEE_UPDATED = 'FEES/DEFAULT_FEE_UPDATED',
+}
+
+export enum FeeType {
+  INVITE = 'invite',
+  SEND = 'send',
+  EXCHANGE = 'exchange',
+  RECLAIM_ESCROW = 'reclaim-escrow',
+}
+
+export interface UpdateDefaultFeeAction {
```

Comment:
The term `Default` is confusing me a bit here. What do you mean by default fee?

Reply:
I agree. I think Estimate is appropriate.
What do you think?

Reply:
Estimate sgtm